jruby/neo4j/driver/ext/internal/summary/internal_result_summary.rb

```diff
@@ -9,6 +9,21 @@ module InternalResultSummary
             %i[result_available_after result_consumed_after].each do |method|
               define_method(method) { super(Java::JavaUtilConcurrent::TimeUnit::MILLISECONDS) }
             end
+
+            def self.query_type(type)
```

Comment:
this method was supposed to override the java method and that's not its signature